Description: Adjust hardcoded library install path to match our actual install
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2012-12-26

--- a/flowblade
+++ b/flowblade
@@ -34,7 +34,7 @@
 # Update sys.path to include modules
 if launch_dir == "/usr/bin":
     print "Running from installation..."
-    modules_path = "/usr/share/pyshared/Flowblade"
+    modules_path = "/usr/share/flowblade/Flowblade"
 else:
     print "Running from filesystem..."
     modules_path = launch_dir + "/Flowblade"
